src/store/messages/getters.ts

```diff
@@ -0,0 +1,17 @@
+import { GetterTree } from 'vuex';
```

Comment:
Aside from the naming concerns, I'm not entirely convinced we need to introduce getters for data that isn't accessed anywhere yet. They're there for convenience, not a necessary part of a store module.